opensrp-reveal/src/main/java/org/smartregister/reveal/view/OfflineMapsActivity.java

```diff
@@ -0,0 +1,61 @@
+package org.smartregister.reveal.view;
+
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.design.widget.TabLayout;
+import android.support.v4.view.ViewPager;
+import android.support.v7.app.AppCompatActivity;
+
+import org.smartregister.family.adapter.ViewPagerAdapter;
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.contract.OfflineMapDownloadCallback;
+import org.smartregister.reveal.contract.OfflineMapsContract;
+import org.smartregister.reveal.fragment.AvailableOfflineMapsFragment;
+import org.smartregister.reveal.fragment.DownloadedOfflineMapsFragment;
+import org.smartregister.reveal.model.OfflineMapModel;
+
+public class OfflineMapsActivity extends AppCompatActivity implements OfflineMapsContract.View, OfflineMapDownloadCallback {
+
+    private AvailableOfflineMapsFragment availableOfflineMapsFragment;
+
+    private DownloadedOfflineMapsFragment downloadedOfflineMapsFragment;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'downloadedOfflineMapsFragment' could be replaced by a local variable.](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)